frontend/src/components/LandingPage.tsx

```diff
@@ -0,0 +1,150 @@
+      (connectWalletDialogOpen) => !connectWalletDialogOpen
+    );
+  };
+
+  const selectWallet = (walletName: string) => {
+    tryConnectWallet(walletName);
+    handleClose();
+  };
+
+  const tryConnectWallet = (walletName: string) => {
+    dispatch(
+      establishWalletConnection({
+        walletName,
+        networks: [...networks, ...getLocalNetworks()],
+      })
+    );
+  };
+
+  useEffect(() => {
+    const walletName = getWalletName();
+    if (isConnected()) {
+      tryConnectWallet(walletName);
+    }
+
+    const accountChangeListener = () => {
+      setTimeout(() => tryConnectWallet(walletName), 1000);
+      window.location.reload();
+    };
+
+    window.addEventListener(
+      `${walletName}_keystorechange`,
+      accountChangeListener
+    );
+
+    dispatch(getAllTokensPrice());
+
+    return () => {
+      window.removeEventListener(
+        `${walletName}_keystorechange`,
+        accountChangeListener
+      );
+    };
+  }, []);
+
+  return connected ? (
+    <>{children}</>
+  ) : (
+    <div>
+      <div className="landingpage-background">
+        <div className="flex justify-center items-center gap-2.5 p-2.5 w-full h-8"></div>
+
+        <div className="pl-20">
+          <Image
+            src="/vitwit-logo-main.png"
+            width={180}
+            height={475}
+            alt="Vitwit-Logo"
+          />
+        </div>
+        <div className="flex items-center gap-10 pl-20 py-0 ">
+          <div className="space-y-40">
+            <div className="flex flex-col space-y-6">
+              <div className="flex text-white text-center text-[100px] not-italic font-extrabold leading-[100px] tracking-[7px]">
+                Resolute
+              </div>
+              <div className="flex flex-col space-y-10">
+                <div className="flex flex-col space-y-2 w-[512px]">
+                  <div className="text-white text-[28px] font-light">
+                    Interface Interchain
+                  </div>
+                  <div className="text-white text-lg font-thin leading-normal">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                    do eiusmod tempor incididun.
+                  </div>
+                </div>
+                <div
+                  className="landingpage-button"
+                  onClick={() => setConnectWalletDialogOpen(true)}
+                >
+                  <p className="text-white text-lg font-bold">Connect Wallet</p>
+                </div>
+              </div>
+            </div>
+
+            <WalletPopup
+              isOpen={connectWalletDialogOpen}
+              onClose={handleClose}
+              selectWallet={selectWallet}
+            />
+          </div>
+
+          <div className="absolute left-[521px] top-[38px]">
+            <Image
+              src="/background-circle.png"
+              width={402}
+              height={456}
+              alt="background-circle"
+            />
```

Comment:
Write media queries for this image. Vertical scroll is appearing on smaller screens.